semantics.json

```diff
@@ -51,6 +51,20 @@
           "optional": true,
           "description": "Optional image for the card. (The card may use just an image, just a text or both)"
         },
+		{
+          "name": "back_image",
+          "type": "image",
+          "label": "Back Image",
+          "optional": true,
+          "description": "Optional image for the back of card. (If this is empty but card on front is specified, that same card will be used for both front & back but not vice versa.)"
+        },
+		{
+		  "name": "use_same",
```

Comment:
I don't see the need for this option. If back_image is provided back image will be used on the backside of the dialog card, otherwise front image will be used on both sides. There should be no need for this option